src/play.c

```diff
@@ -302,8 +306,12 @@ int play_tone(struct play **playp, struct player *player,
 	err = auplay_alloc(&play->auplay, baresip_auplayl(),
 			   play_mod, &wprm,
 			   play_dev, write_handler, play);
-	if (err)
+       if (err) {
```

Comment:
the indentation looks wrong (spaces, instead tab), not sure why linting is not complaining.